### DISCUSSION TITLE
How to set default plugin to Datagrid

### DISCUSSION THREAD

**Discussion by akash-goel** [perspective-dev/perspective#2005] [Q&A]
Hi Team,

We are using Perspective Viewer , When its loading by default its first loading the X-bar and when we are doing the reload with Plugin Datagrid than its pointing to Datagrid , When we are calling reset functionality in Perspective Viewer than also by default its showing X-Bar .

Can you please guide how we can set the default plugin to Datagrid only.

**Comment by timkpaine** [comment]
did you include the data grid plugin in your project? https://perspective.finos.org/docs/js#which-modules-should-i-import
